zigzag/zigzag_test_log.py

```diff
@@ -599,16 +662,124 @@ def __init__(self, testcase_name, teststeps_xml, mediator):
                 a single qTest test log.
             mediator (ZigZag): The mediator that stores shared data.
         """
-
-        # Initialize without parsing so we can post-process the test steps.
-        super(_ZigZagTestLogWithSteps, self).__init__(None, mediator, auto_parse=False)
+        super(_ZigZagTestLogWithSteps, self).__init__(None, mediator)
```

Comment:
It just dawned on me that we can just send send the first `teststeps_xml` element to the parent class initializer and skip having to override a bunch of properties.

```suggestion
        super(_ZigZagTestLogWithSteps, self).__init__(teststeps_xml[0], mediator)  # All steps share similar info
```